public/pages/HistoricalDetectorList/components/EmptyHistoricalDetectorMessage/EmptyHistoricalDetectorMessage.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { EuiButton, EuiEmptyPrompt, EuiText } from '@elastic/eui';
+import React from 'react';
+import { APP_PATH, PLUGIN_NAME } from '../../../../utils/constants';
+
+const filterText =
```

Comment:
better to have constant variable name in UPPER_UNDERSCORE_CASE

Reply:
good catch. Copied from existing `DetectorList`'s `EmptyMessage` component. Will change there as well.